src/dbnode/integration/index_single_node_high_concurrency_test.go

```diff
@@ -258,6 +258,7 @@ func testIndexSingleNodeHighConcurrency(
 			zap.Int("concurrency", opts.concurrencyQueryDuringWrites))
 		checkNumTotalQueryMatches = true
 		for i := 0; i < opts.concurrencyQueryDuringWrites; i++ {
+			i := i
```

Comment:
Heh, good catch. Wish there was a linter for this...

Reply:
VSCode linter was complaining about this luckily :)